src/tests/unit10Tests.spec.js

```diff
@@ -0,0 +1,31 @@
+const { expect } = require('@playwright/test');
```

Comment:
Let's rename the file name, i.e. sortingOnTheInventoryPage.spec.js.
This title would be more appropriate.